screens/Ingredients/IndexIngredientScreen.js

```diff
@@ -16,20 +19,95 @@ import formatMoney from '../../utils/formatMoney';
 
 function IndexIngredients({ navigation }) {
   const getIngredients = useStoreActions((actions) => actions.getIngredients);
+  const setInventory = useStoreActions((actions) => actions.setIngredientInventory);
 
   const [ingredients, setIngredients] = useState([]);
   const evenNumber = 2;
+  const [editableInventories, setEditableInventories] = useState([]);
+
+  const increaseInventory = (ingId) => {
+    const newIngredients = [...ingredients];
+
+    newIngredients.map((ing) => {
+      if (ing.id == ingId & ing.attributes.inventory >= 0) {
+        ing.attributes.inventory = ing.attributes.inventory + 1;
+      }
+    });
+    setIngredients(newIngredients);
+  };
+
+  const decreaseInventory = (ingId) => {
+    const newIngredients = [...ingredients];
+
+    newIngredients.map((ing) => {
+      if (ing.id == ingId & ing.attributes.inventory > 0) {
+        ing.attributes.inventory = ing.attributes.inventory - 1;
+      }
+    });
+    setIngredients(newIngredients);
+  };
+
+  const showInventoryInput = (index) => {
+    const copyEditables = [...editableInventories];
+    if (copyEditables[index]) {
+      copyEditables[index] = false;
+    } else {
+      copyEditables[index] = true;
+      submitInventoryValue(ingredients[index]);
+    }
+    setEditableInventories(copyEditables);
+  };
+
+  const setInventoryInIngredient = (event, ingId) => {
+    const newIngredients = [...ingredients];
+
+    newIngredients.map((ing) => {
+      if (ing.id == ingId & ing.attributes.inventory >= 0) {
+        ing.attributes.inventory = Number(event);
+      }
+    });
+    setIngredients(newIngredients);
+  };
+
+  const submitInventoryValue = async (ingredient) => {
+    const attributes = {
+      // eslint-disable-next-line
+      provider_name: ingredient.attributes.provider_name,
+      name: ingredient.attributes.name,
+      sku: ingredient.attributes.sku,
+      price: ingredient.attributes.price,
+      currency: ingredient.attributes.currency,
+      quantity: ingredient.attributes.quantity,
+      measure: ingredient.attributes.measure,
+      inventory: ingredient.attributes.inventory,
+      // eslint-disable-next-line
+      ingredient_measures_attributes: formatIngredientMeasure(ingredient.attributes.other_measures)
+    };
```

Comment:
Por qué si lo único que se quiere cambiar es el `inventory` se tiene que enviar todas las demás cosas?

Reply:
@lldavico  está en pr en backend (pero ya aprobado) una forma de actualizar el inventario que sólo necesita el id del ingrediente y el valor actualizado! Esperaría el merge de eso y hacemos un refactor. Se va a simplificar mucho el código y se resuelve este problemm!!

Reply:
this https://github.com/platanus/2021-1-Grupo4-Web/pull/120

Reply:
Oka! esperemos a eso. Por mientras revisaré los otros comentarios para hacer los cambios en el código

Reply:
Cambiado